typescript/core/src/McapIndexedReader.test.ts

```diff
@@ -130,6 +130,7 @@ describe("McapIndexedReader", () => {
         ...string(""), // profile
         ...string(""), // library
       ]),
+      ...record(Opcode.DATA_END, [...uint32LE(0)]),
```

Comment:
Previously the reader wouldn't detect a missing `DataEnd` record and now it does?

Reply:
yep!